tests/ops/fixtures/saas/datadog_fixtures.py

```diff
@@ -95,3 +95,70 @@ def datadog_dataset_config(
     )
     yield dataset
     dataset.delete(db=db)
+
+
+@pytest.fixture(scope='session')
+def generate_test_data_on_datadog(*args, **kwargs):
```

Comment:
Let's continue our naming convention of `datadog_access_data`, we also don't need `*args`, `**kwargs` here.

Reply:
Also, make sure you include the fixture as part of `test_saas_access_request_task` so this can actually run.